### ISSUE TITLE
Change for compatibility with 0.8.5

### ISSUE THREAD

**Issue by davidR1974** [crash1115/blind-roll-skills#24]
Hello,

i was having a look at modulecompatility for 0.8.5 and it seems it work by replacing (in blindroll.js / line 52-56)

```
    if(msg.flags){
      if(skillsToBlind.includes(msg.flags.dnd5e?.roll?.skillId)) { makeRollBlind = true; }
      if(game.settings.get("blind-roll-skills", "hideDeathSaves") && (msg.flags.dnd5e?.roll?.type === "death")) { makeRollBlind = true; }
      if(game.settings.get("blind-roll-skills", "hideInitiative") && msg.flags['core.initiativeRoll']) { makeRollBlind = true; }
    }
```

with

```
    if(msg.data.flags){
      if(skillsToBlind.includes(msg.data.flags.dnd5e?.roll?.skillId)) { makeRollBlind = true; }
      if(game.settings.get("blind-roll-skills", "hideDeathSaves") && (msg.data.flags.dnd5e?.roll?.type === "death")) { makeRollBlind = true; }
      if(game.settings.get("blind-roll-skills", "hideInitiative") && msg.data.flags['core.initiativeRoll']) { makeRollBlind = true; }
    }
```

Hope that helps if you did not look into it yet.

**Comment by crash1115** [comment]
Hey, thanks for this. Updating is lagging behind a bit, as I'm waiting for some of the supported roll modules (like Better Rolls) to get themselves stabilized on the new version, but I'll try to be on top of it shortly.

**Comment by Avitale34** [comment]
Came here for some reason! Love the module look forward to the release :)

**Comment by crash1115** [comment]
Currently I'm waiting on an issue with how Core displays the content of blind/whisper/private rolls to get this finished up. There was a change made way back in 0.8.1 that messed things up a little bit and it didn't get discovered until like last week.

It seems like a fix for that will be out next week sometime? I'm keeping an eye on it for sure. https://gitlab.com/foundrynet/foundryvtt/-/issues/5302

**Comment by crash1115** [comment]
Fixed in 0.7.1.
